declarations/Zoosk.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Zoosk",
+  "importedFrom": "https://github.com/tosdr/tosback2/blob/f762cd4bbb2571272985fed009201a23300ba9b2/rules/zoosk.com.xml",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://docviewer.zoosk.com/legal-tos-en_eu.html",
+      "select": ".terms-of-service-eu"
+    },
+    "Privacy Policy": {
+      "fetch": "https://docviewer.zoosk.com/legal-privacy-en_eu.html",
+      "select": ".legal__inner"
```

Comment:
I removed the cookie settings button, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/f2f90904dd93f5328530df236809ba14b90f007e